site/content/api/Event-API.md

```diff
@@ -76,4 +81,82 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+NB: currently there is only one such *well-known* event.
+
+### Lifecycle Notifications
+
+A specific event can signal that a device is `connected` (e.g. to signal that it is ready to receive a command for a specific time interval) 
```

Comment:
FMPOV it is not the event that signals anything. However, the event can be used by other components to signal that a device has connected to a protocol adapter. My understanding was that we agreed to distinguish between the more technical events for signalling that a device has connected/disconnected and the *flag* that devices use to signal that they are now ready to receive a command. IMHO we should not start to mix them up again by indicating that the *connected* event is used to signal that a device is ready to receive a command. FMPOV this is not the case. This event should simply indicate that a device has connected, nothing more, nothing less.

Reply:
After having read your other comments I think I see your concern here:

I think we agreed on ONE event, but two different events make some sense, I agree (see my comments below).
Will rephrase the wording that it is not the *event* that is signaling something itself (how should that be?), but it can be used by others to signal connection/disconnection. And remove the example in brackets here, so there is no reference to command readiness anymore.